packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -24,7 +24,10 @@ class WebpackCLI {
 
     async makeCommand(commandOptions, options, action) {
         const alreadyLoaded = this.program.commands.find(
-            (command) => command.name() === commandOptions.name || command.aliases().includes(commandOptions.alias),
+            (command) =>
+                command.name() === commandOptions.name ||
+                command.aliases().includes(commandOptions.alias) ||
+                JSON.stringify(command.aliases()) === JSON.stringify(commandOptions.alias),
```

Comment:
Bad for performance

Reply:
We have command.name() = "init",
commandOptions.name = "init [generation-path]"

How about the following? 

```suggestion
                command.name() === commandOptions.name.split(' ')[0],
```